activiti-spring-boot-starter/src/test/java/org/activiti/spring/boot/RuntimeTestConfiguration.java

```diff
@@ -221,7 +222,9 @@ public Connector variableMappingActionName() {
                             tuple(variableThree,
                                   5),
                             tuple(unmappedMatchingVariable,
-                                  "inTest"));
+                                  "inTest"),
```

Comment:
@balsarori note that this connector has a connector definition file. In addition to this test, I think it would be judicious to add another one where the connector has no connector definition (as this is the case you want to cover).

Reply:
@balsarori I've added the test.

Reply:
Thanks